netlib/debug.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import (absolute_import, print_function, division)
+
+import sys
+import threading
+import signal
+import platform
+
+import psutil
+
+from netlib import version
+
+
+def sysinfo():
+    data = [
+        "Mitmproxy verison: %s" % version.VERSION,
+        "Python version: %s" % platform.python_version(),
+        "Platform: %s" % platform.platform(),
```

Comment:
Can we add the openssl version as well?
`from OpenSSL import SSL; print(SSL.SSLeay_version(SSL.SSLEAY_VERSION))`